memory/memhotplug.py

```diff
@@ -100,14 +100,19 @@ def setUp(self):
         if not memory.check_hotplug():
             self.cancel("UnSupported : memory hotplug not enabled\n")
         smm = SoftwareManager()
+        for package in ['automake', 'make', 'autoconf']:
```

Comment:
is needed when the user wants to run upstream (defined url) ? please move this when stress package not installed

Reply:
made necessary changes